docs/en_us/platform_api/source/links.rst

```diff
@@ -1 +1,7 @@
-.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
\ No newline at end of file
+.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
+
+.. _edX platform documentation: http://docs.edx.org/
```

Comment:
This is the links.rst file in the edx-platform repository, so it's separate from our usual one.